src/components/overlay/overlay.tsx

```diff
@@ -140,7 +140,7 @@ class Overlay extends Component<any, any> {
     }
 
     return (
-      <div tabIndex={-1} className={overlayClass.join(' ')} role="dialog" onKeyDown={this.onKeyDown} aria-label={label}>
+      <div tabIndex={-1} className={overlayClass.join(' ')} role="dialog" onKeyDown={this.onKeyDown} aria-labelledby="captionsDialogTitle">
```

Comment:
does tooltip still work if aria-label is removed ?

Reply:
Yes, works the same as before.